### ISSUE TITLE
Postgis @variables error

### ISSUE THREAD

**Issue by springmeyer** [mapnik/node-mapnik#882]
Seeing:

```
  1) mapnik.VectorTile postgis.input passes variables to replace tokens in query:
     Uncaught Error: Postgis Plugin: ERROR:  column "fieldid" does not exist
LINE 1: ...T * FROM (SELECT gid, geom FROM test WHERE gid = @fieldid) a...
                                                             ^
in executeQuery Full sql was: 'SELECT * FROM (SELECT gid, geom FROM test WHERE gid = @fieldid) as field_shapes LIMIT 0'
  encountered during parsing of layer 'field_shapes' in Layer of './test/data/postgis_datasource_tokens_query.xml'
      at Error (native)
      at ChildProcess.<anonymous> (test/vector-tile.postgis.test.js:67:21)
      at Process.ChildProcess._handle.onexit (internal/child_process.js:211:12)
```

On travis: https://travis-ci.org/mapnik/node-mapnik/jobs/396403731#L2544

This looks like a problem with https://github.com/mapnik/node-mapnik/pull/809.

/cc @rafatower @Algunenano - any idea what might be causing this error?

**Comment by Algunenano** [comment]
I think that the issue is that mapnik 3.1 changes how the variables are declared inside sql queries, so it should be [`|@fieldid|`](https://github.com/CartoDB/node-mapnik/blob/v3.6.2-carto/test/data/postgis_datasource_tokens_query.xml#L11).

**Comment by springmeyer** [comment]
Ah right, thank you @Algunenano. I completely forgot about that change. Reminder to self: https://github.com/mapnik/mapnik/wiki/API-changes-between-v3.0-and-v3.1#changed | https://github.com/mapnik/mapnik/pull/3618.

So, I guess we'll need to adapt the tests here in master.

**Comment by springmeyer** [comment]
solved by 18a4a0e, thanks again for the tip @Algunenano
